src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2237,6 +2237,49 @@
      * The {@code default} keyword.  This element has no
      * children.
      *
+     * <p>For example:</p>
+     * <pre>
+     * switch (3){
```

Comment:
I have changed your example a little bit, lets use it:

```
     * switch (type) {
     *   case 1:
     *     x = 1;
     *     break;
     *   default:
     *     x = 3;
     * }
```

Reply:
Ok I didn't notice. Done this too @timurt